rll/metadata.rb

```diff
@@ -10,6 +10,7 @@
 recipe      "rll::upgrade", "Check whether a RightLink upgrade is available and do the upgrade"
 recipe      "rll::test-script", "Test operational script, doesn't do anything useful"
 recipe      "rll::shutdown-reason", "Print out the reason for shutdown"
+recipe      "rll::automate_upgrade", "Periodically checks if an upgrade is available and upgrade if there is."
```

Comment:
Is this really the name we want for this? It seems like we'd want to call it `automatic_upgrade` or `setup_automatic_upgrade`.